### ISSUE TITLE
Stimulum gas never gives its benefits, or borderline so.

### ISSUE THREAD

**Issue by Unit2E** [tgstation/tgstation#41186]
Issue reported from  Round ID: 96141 (/tg/Station Sybil [ENGLISH] [US-EAST] [100% LAG FREE])

Reporting client version: 512

Plainly, stimulum doesn't work. In this particular round, I made stimulum to see what it was like, or if it worked even. Turns out that, no, it does not work (alternatively, perhaps it does not work when using pluoxium as well.)

As for some more exact stats, and what actually happens. I added 600 kpA pluoxium to an empty oxygen tank. After this, I filled the remaining volume up with stimulum, up to 1013 kPa. After this, at 16 kPa release, Stimulum was very rarely added to the blood stream, or not at all. When it does get added, it nearly instantly disappears, seemingly having little effect as well, or perhaps none? When I raised the release pressure to 200 kPa on the tank, Stimulum gets added more regularly added to the bloodstream seemingly, but, it still practically instantly disappears with little effect. I'm unsure what causes this, but in any case, stimulum is not usable in its current state.

In short, when breathing stimulum it adds 1 unit of it to the bloodstream, after which it disappears fairly quickly, probably next tick. This leaves huge gaps of stimulum being in the bloodstream, spending most time without it in your bloodstream even if breathing it from a tank with about 50% of it in there with high release pressure.

After this, I tested other makeups of the internals. 100% pure stimulum seems to keep it in your bloodstream permanently, but you can't breathe that. Seems it's oddly tied to the concentration of what you're breathing.

pls fix.
